content/docs/examples/kubeflow-samples.md

```diff
@@ -8,46 +8,4 @@ weight = 10
 This section introduces the examples in the
 [kubeflow/examples](https://github.com/kubeflow/examples) repo.
```

Comment:
This link is still broken. See preview: https://deploy-preview-1620--competent-brattain-de2d6d.netlify.com/docs/examples/kubeflow-samples/

Reply:
I saw this as well, and it happens on the Master branch. But the link shows up correct on Github Markup and when I build-docs locally. I think it might be a bug but will have to investigate. Do you suggest we do it as part of this review or on a separate issue?

Reply:
Hmm the link is working on the master branch: https://master.kubeflow.org/docs/examples/kubeflow-samples/

It'd be nice if we can fix this before merging the PR. Here are some suggestions to try:

* Try changing from Markdown to HTML for the link. (You can still use a relative URL even in the HTML.
* If that doesn't work, I wonder if the has something to do with the fact that you're using two different types of content block? I don't think that should be a problem, but you could try changing the introductory paragraph to just plain Markdown, not in a block. Or you could put it in one of your new `samples-section` blocks, and make sure the `samples-section` block works even if you pass it no parameters.

Reply:
Thanks for fixing it!